Woof/WoofTests/ViewModel/OwnerMainTabViewModelTests.swift

```diff
@@ -1,18 +1,56 @@
 import XCTest
 
 final class OwnerMainTabViewModelTests: XCTestCase {
-    func testResetCurrentRoleMethodExistsInAPI() {
-        OwnerMainTabViewModel().resetCurrentRole()
+    override func setUp() {
+        super.setUp()
+        viewModel = OwnerMainTabViewModel()
     }
 
-    func testResetCurrentRoleChangesUserRoleToNone() {
-        // Given
-        PreferencesHandler.set(userRole: .owner)
+    func testIsAlertShownPropertyExistsInAPI() {
+        _ = viewModel.isAlertShown
+    }
 
-        // When
-        OwnerMainTabViewModel().resetCurrentRole()
+    func testIsAlertShownPropertyReturnsAssignedValue() {
+        // Given // When
+        viewModel.isAlertShown = true
 
         // Then
-        XCTAssertEqual(PreferencesHandler.getUserRole(), Role.none)
+        XCTAssertTrue(viewModel.isAlertShown)
+
+        // Given // When
+        viewModel.isAlertShown = false
+
+        // Then
+        XCTAssertFalse(viewModel.isAlertShown)
+    }
+
+    func testIsAlertShownPropertyReturnsFalseByDefault() {
+        XCTAssertFalse(viewModel.isAlertShown)
+    }
+
+    func testIsLogoutConfirmedPropertyExistsInAPI() {
+        _ = viewModel.isLogoutConfirmed
     }
+
+    func testIsLogoutConfirmedPropertyReturnsAssignedValue() {
+        // Given // When
```

Comment:
Nit: maybe  we  can remove this section:
        // Given // When //Then
And combine all Asserts together?

Reply:
I think it should be left to stick to a unified style of writing tests.